docs/backend/schema.ddl

```diff
@@ -180,6 +155,7 @@ Table SequencingProtocol {
 }
 
 // describes a single set of sequence data from the sequencer. (FASTQ)
+// these should be replaced by the `HostFilteredSequence`as soon as it is available so we are not storing any host reads
```

Comment:
For consistency, I would also rename `HostFilteredSequence` to `HostFilteredSequencingReads`

Reply:
I have done this rename in https://github.com/chanzuckerberg/aspen/pull/79/